packages/ra-ui-materialui/src/list/datagrid/Datagrid.stories.tsx

```diff
@@ -585,3 +587,198 @@ export const LabelElements = () => (
         </AdminContext>
     </TestMemoryRouter>
 );
+
+const AccessControlUI = ({
+    children,
+    setAuthorizedResources,
+    authorizedResources,
+    queryClient,
+}: {
+    children: React.ReactNode;
+    setAuthorizedResources: Function;
+    authorizedResources: {
+        books: boolean;
+        'books.id': boolean;
+        'books.title': boolean;
+        'books.author': boolean;
+        'books.year': boolean;
+    };
+    queryClient: QueryClient;
+}) => {
+    return (
+        <div>
+            <div>
+                <label>
+                    <input
```

Comment:
unchecking this checkbox has no effect. Is it normal?

Reply:
Removed the checkbox as it made no sense here